app/client/lib/ethereum/observeBlocks.js

```diff
@@ -19,49 +19,64 @@ Update wallet balances
 @method updateBalances
 */
 updateBalances = function() {
+    // UPDATE TOKEN BALANCES
```

Comment:
Account balances are updated using the EthAccount package. Doing it here too would be double the call amounts.